CODEOWNERS

```diff
@@ -2,4 +2,4 @@
 # This file contains a list of project codeowners
 #
 
-* @UCLALibrary/services-team-reviewers
```

Comment:
Let's go ahead and leave this with services-team-reviewers... I thought that wasn't working well enough at one point and switched it back to individual names but have since concluded it's fine.